package.json

```diff
@@ -100,7 +100,7 @@
   },
   "devDependencies": {
     "@babel/core": "^7.23.2",
-    "@babel/eslint-parser": "^7.22.15",
+    "@babel/eslint-parser": "^7.11.0",
```

Comment:
Hope this package was downgraded by mistake.

Reply:
my yarn install must have downgraded it, will revert